Update Harvester v1.0 install configs (#98)
harvester/docs: docs/rancher/rancher-integration.md

[Rancher](https://github.com/rancher/rancher) is an open-source multi-cluster management platform. Harvester has integrated Rancher by default starting with Rancher v2.6.1.

!!!note
    Harvester v1.0.0 is compatible with Rancher v2.6.3 or above only.

Users can now import and manage multiple Harvester clusters using the Rancher [Virtualization Management](virtualization-management.md) page and leverage the Rancher [authentication](https://rancher.com/docs/rancher/v2.6/en/admin-settings/authentication/) feature and RBAC control for [multi-tenancy](https://rancher.com/docs/rancher/v2.6/en/admin-settings/rbac/) support.

![virtualization-management](assets/virtualization-management.png)

## Deploying Rancher

> To use Rancher with Harvester, please install the Rancher server separately from the Harvester. As an option, You can spin up a VM in the Harvester and install the Rancher v2.6.3 or above to try out the integration features.

Reviewer: ```suggestion
To use Rancher with Harvester, please install the Rancher and Harvester in two separated servers. If you want to try out the integration features, you can create a VM in Harvester and install Rancher v2.6.3 or above.
```

```shell
# for a quick evaluation, you can run the Rancher server with the following command
$ sudo docker run -d --restart=unless-stopped -p 80:80 -p 443:443 --privileged rancher/rancher:v2.6.3
```

!!! Note

>     For more information about how to deploy the Rancher server, please refer to the Rancher [documentation](https://rancher.com/docs/rancher/v2.6/en/quick-start-guide/deployment/).

Reviewer: ```suggestion
    For more information on how to deploy the Rancher server, please refer to the Rancher [documentation](https://rancher.com/docs/rancher/v2.6/en/quick-start-guide/deployment/).
```

## Virtualization Management

> With Rancher's Virtualization Management feature, users can now import and manage Harvester clusters. 

Reviewer: ```suggestion
With Rancher's Virtualization Management feature, you can now import and manage Harvester clusters. 
```

> By clicking on one of the clusters, users are able to view and manage the imported Harvester cluster resources like Hosts, VMs, images, volumes, etc. Additionally, the `Virtualization Management` has leveraged existing Rancher features such as authentication with various auth providers and multi-tenant support.

Reviewer: ```suggestion
By clicking on one of the clusters, you are able to view and manage the imported Harvester cluster resources like Hosts, VMs, images, volumes, etc. Additionally, the `Virtualization Management` leverages existing Rancher features such as authentication with various auth providers and multi-tenant support.
```
